src/Components/Artwork/Details/PrimaryLabelLine.tsx

```diff
@@ -14,21 +14,15 @@ interface PrimaryLabelLineProps {
   me?: PrimaryLabelLine_me$data
 }
 
-const doesShipFree = (
-  artwork: PrimaryLabelLine_artwork$data,
-  me?: PrimaryLabelLine_me$data,
-): boolean => {
-  const result = useFulfillmentOptionsForArtwork(artwork, me)
-  const { freeShippingToUserCountry, freeGlobalShipping } = result || {}
-  return !!(freeShippingToUserCountry || freeGlobalShipping)
-}
-
 export const PrimaryLabelLine: React.FC<
   React.PropsWithChildren<PrimaryLabelLineProps>
 > = ({ label, artwork, me }) => {
   const { hideSignals, updateSignals } = useArtworkGridContext()
   const partnerOffer = artwork?.collectorSignals?.partnerOffer
-  const shipsFree = artwork && doesShipFree(artwork, me)
+
+  const { freeShippingToUserCountry, freeGlobalShipping } =
+    processArtworkFulfillmentOptions(artwork, me) || {}
+  const shipsFree = freeShippingToUserCountry || freeGlobalShipping
```

Comment:
Not trying to be finnicky here with the feedback, but just to note that the only reason this complexity is necessary is due to destructuring. It can be simplified (we don't always have to destructure):

```tsx
const fulfillmentOptions = processArtworkFulfillmentOptions(artwork, me)

const shipsFree = fulfillmentOptions?.freeShippingToUserCountry || fulfilmentOptions.freeGlobalShipping 
```